Add duplication as url query to all sessions

aronerben opened the pull request:
Simply adds a button for all sessions to duplicate that session. This opens a new form with the prefilled values. This new form will always create a parent session, regardless if the chosen-to-duplicate session was a follow-up session.

timohuber commented:
@aronerben Was this a requirement, that you cannot duplicate follow-up sessions? Wouldn't that feature be implemented rather quickly, wouldn't it?

And somehow there is a merge conflict. Please resolve.

aronerben commented:
> @aronerben Was this a requirement, that you cannot duplicate follow-up sessions? Wouldn't that feature be implemented rather quickly, wouldn't it?
> 
> And somehow there is a merge conflict. Please resolve.

@timohuber Thanks for the hint, I added the follow-up duplication and fixed the merge conflict.
